### PR TITLE
update arg_parser.py to fix `--port` argument type

### PR DESCRIPTION
**Pull request by lfyzjck**
`--port` argument defualt type is `str`, this will cause a consul register error.

### CODE REVIEW

File: src/ianitor/args_parser.py [hunk 1]
```diff
@@ -143,6 +143,7 @@ def get_parser():
 
     parser.add_argument(
         "--port",
+        type=int, default=0,
```
Comment (pwilczynskiclearcode) [inline src/ianitor/args_parser.py]: default `0` ? I think it's not even possible to use such port.

Comment (lfyzjck) [inline src/ianitor/args_parser.py]: perhaps there is no default value here, argument `--port` still equals to `0` even leave it empty.

Comment (swistakm) [inline src/ianitor/args_parser.py]: 0 to 1023 are "well known ports" but there is no document that says 0 cannot be used and it has no official protocol assigned for neither TCP nor UDP. 0 is sometimes used (unofficialy) for specifying system allocated ports

**Comment by swistakm** [comment]
build failed due to #4

**Comment by swistakm** [comment]
someone could retry the build manually. I don't have such access rights anymore.

**Comment by lfyzjck** [comment]
can't reappear this test failure in local environment.

**Comment by pwilczynskiclearcode** [comment]
> someone could retry the build manually. I don't have such access rights anymore.

I granted you access to ianitor back again.

**Comment by swistakm** [comment]
Merged and released on PyPI as `0.0.3`.

@pwilczynskiclearcode thanks for access grant!
@lfyzjck thanks for your contribution!

**Pull request merged**